src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -94,6 +95,9 @@
     <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
       <Link>ProductionCode\Common\System\Threading\Tasks\TaskToApm.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\System\PinnableBufferCache.cs">
```

Comment:
Please do not include production code in Functional Tests.

FunctionalTests = e2e using public surface only
UnitTests = testing a single class/method, we achieve this by including production code within the test